internal/volumes/volumes.go

```diff
@@ -248,6 +249,7 @@ func GetBindMount(sys *types.SystemContext, args []string, contextDir string, st
 		if additionalMountPoints != nil {
 			if val, ok := additionalMountPoints[fromWhere]; ok {
 				mountPoint = val.MountPoint
+				skipOverlay = val.IsWritesDiscardedOverlay
```

Comment:
Isn’t this ~always `false` because `fromWhere != ""`? OTOH maybe the consistency is nice.

Reply:
Oh, right, we haven't mounted an overlay there yet when it's something else.  You're reading it right.